.travis.yml

```diff
@@ -39,9 +39,9 @@ matrix:
 
   - node_js: 8
     env:
-      - TEST_SUITE=build
+      - TEST_SUITE=dist
```

Comment:
I would suggest leaving this name as-is. Otherwise you also need to update travis-env-vars.sh.